test/browser.multiapp.test.js

```diff
@@ -0,0 +1,100 @@
+var boot = require('../');
+var exportBrowserifyToFile = require('./helpers/browserify').exportToFile;
+var fs = require('fs');
+var path = require('path');
+var expect = require('chai').expect;
+var browserify = require('browserify');
+var sandbox = require('./helpers/sandbox');
+var vm = require('vm');
+var createBrowserLikeContext = require('./helpers/browser').createContext;
+var printContextLogs = require('./helpers/browser').printContextLogs;
+
+describe('browser support for multiple apps', function() {
+  this.timeout(60000); // 60s to give browserify enough time to finish
+
+  beforeEach(sandbox.reset);
+
+  it('has API for bundling and booting multiple apps', function(done) {
+    var app1Dir = path.resolve(__dirname, './fixtures/browser-app');
+    var app2Dir = path.resolve(__dirname, './fixtures/browser-app-2');
+
+    var apps = [
+      {
+        appDir: app1Dir,
+        appFile: './app.js',
+        moduleName: 'browser-app'
+      },
+      {
+        appDir: app2Dir,
+        appFile: './app.js',
+        moduleName: 'browser-app2',
+        appId: 'browserApp2'
+      }
+    ];
+
+    browserifyTestApp(apps, function(err, bundlePath) {
+      if (err) return done(err);
+
+      var bundledApps = executeBundledApp(bundlePath, apps);
+      var app1 = bundledApps.defaultApp;
+      var app2 = bundledApps.browserApp2;
+
+      expect(app1.settings).to.have.property('custom-key', 'custom-value');
+      expect(Object.keys(app1.models)).to.include('Customer');
+      expect(Object.keys(app1.models)).to.not.include('Robot');
+      expect(app1.models.Customer.settings).to.have.property('_customized',
+        'Customer');
+
+      expect(Object.keys(app2.models)).to.include('Robot');
+      expect(Object.keys(app2.models)).to.not.include('Customer');
+
+      done();
+    });
+  });
+});
+function browserifyTestApp(apps, next) {
```

Comment:
`browserifyTestApps` (plural), add an empty line between L54-L55 to visually delimit functions.